app/Listeners/ProjectMembers.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Listeners;
+
+use App\GenericModel;
+use App\Profile;
+use Illuminate\Support\Facades\Config;
+use Vluzrmos\SlackApi\Facades\SlackChat;
+
+class ProjectMembers
+{
+    /**
+     * Handle the event
+     * @param \App\Events\ProjectMembers $event
+     */
+    public function handle(\App\Events\ProjectMembers $event)
+    {
+        $project = $event->model;
+
+        if ($project->isDirty()) {
+            $oldFields = $project->getOriginal();
+            $updatedFields = $project->getDirty();
+            if ($project['collection'] === 'projects' && key_exists('members', $updatedFields)) {
+                //if user is added to project send slack notification
+                foreach ($updatedFields['members'] as $newMemberId) {
+                    if (!in_array($newMemberId, $oldFields['members'])) {
+                        $member = Profile::find($newMemberId);
+                        if ($member->slack) {
+                            $this->slackMessageUser($member, $project, true);
```

Comment:
replace status value with `self::STATUS_ADDED`, add status value constants to this class